views/users/confirmations/new.php

```diff
@@ -14,32 +16,36 @@
     </div>
   <?php endif; ?>
   <form action="/confirmations/create" method="POST" autocomplete="off">
-    <div>
-      <input type="hidden" id="code" name="code">
-      <input type="hidden" id="token" name="token" value="<?= $data['confirm_token']; ?>">
-      <div id="inputs">
-        <input class="inputs" id="input1" type="text" placeholder="0" maxLength="1" required>
-        <input class="inputs" id="input2" type="text" placeholder="0" maxLength="1" required>
-        <input class="inputs" id="input3" type="text" placeholder="0" maxLength="1" required>
-        <input class="inputs" id="input4" type="text" placeholder="0" maxLength="1" required>
-        <input class="inputs" id="input5" type="text" placeholder="0" maxLength="1" required>
-        <input class="inputs" id="input6" type="text" placeholder="0" maxLength="1" required>
-      </div>
-      <div class="confirm">
-        <input type="submit" class="verify" name="check" value="Confirm">
-      </div>
+    <input type="hidden" id="code" name="code">
+    <input type="hidden" id="token" name="token" value="<?= $data['confirm_token']; ?>">
+    <div id="inputs" class="group-inputs">
+      <input class="inputs" id="input1" type="text" placeholder="0" maxLength="1" required>
+      <input class="inputs" id="input2" type="text" placeholder="0" maxLength="1" required>
+      <input class="inputs" id="input3" type="text" placeholder="0" maxLength="1" required>
+      <input class="inputs" id="input4" type="text" placeholder="0" maxLength="1" required>
+      <input class="inputs" id="input5" type="text" placeholder="0" maxLength="1" required>
+      <input class="inputs" id="input6" type="text" placeholder="0" maxLength="1" required>
     </div>
+      <input type="submit" class="verify" name="check" value="Verify Code">
   </form>
   <div class="resend_code">
-    <p>Didn't receive code?</p>
+    <p>Haven't received it?</p>
     <form action="/confirmations/update" method="POST" autocomplete="off">
       <input type="hidden" id="token" name="token" value="<?= $data['confirm_token']; ?>">
-      <input type="submit" class="resend" name="resend" value="Resend">
+      <input type="submit" class="resend" name="resend" value="Resend code">
     </form>
   </div>
-  <p class="warning">Verification Code is valid only for 10 minutes</p>
+  <p class="warning">The code will only be valid for 10 minutes</p>
 </section>
 <script src="/resources/js/veri_code.js"></script>
+<script>
+  setTimeout(function(){
+    var errorAlert = document.getElementById("error-alert");
+    if (errorAlert) {
+      errorAlert.style.display = "none";
+    }
+  }, 3000);
+</script>
```

Comment:
i last pr #62 the app,sleep function were inplemented, so you can using this in a the js script, instead of making script on php embedded since is a bad practice